### PR TITLE
add Koor Technologies, Inc. as contributor and add support providers list to footer

### PR DESCRIPTION
**Pull request by galexrt**
Signed-off-by: Alexander Trost <galexrt@googlemail.com>

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -32,10 +32,17 @@
...
+        <a href="{{ koorLink }}">Koor Technologies, Inc.</a>
         <a href="{{ redHatLink }}">Red Hat</a>
         <a href="{{ suseLink }}">SUSE</a>
         <a href="{{ upboundLink }}">Upbound</a>
       </div>
+      <div class="col_sm-12">
+        <span>Support Providers</span>
+        <a href="{{ koorLink }}">Koor Technologies, Inc.</a>
+        <a href="{{ redHatLink }}">Red Hat</a>
+        <a href="{{ suseLink }}">SUSE</a>
```
Comment (travisn) [inline _includes/footer.html:44]: I don't think SUSE is supporting it anymore

File: _includes/footer.html [hunk 2]
```diff
@@ -32,10 +32,17 @@
         <span>Top Contributors</span>
         <a href="{{ cloudicalLink }}">Cloudical</a>
         <a href="{{ cybozuLink }}">Cybozu, Inc</a>
+        <a href="{{ koorLink }}">Koor Technologies, Inc.</a>
         <a href="{{ redHatLink }}">Red Hat</a>
         <a href="{{ suseLink }}">SUSE</a>
         <a href="{{ upboundLink }}">Upbound</a>
       </div>
+      <div class="col_sm-12">
```
Comment (travisn) [inline _includes/footer.html:40]: Let's split this commit into a separate PR so it unblocks the other commit for merge. We need to discuss with maintainers (e.g. in the community meeting) if this is the right place to expose this info.

**Review by travisn — CHANGES_REQUESTED** [review]

**Review by travisn — APPROVED** [review]

File: _includes/footer.html (hunk 2 above)
Comment (galexrt) [reply _includes/footer.html]: Done, I have created #131
